examples/miscellanea/simpsons-monorail-map.py

```diff
@@ -0,0 +1,136 @@
+------------------------------------------------
+
+This example demonstrates how to create a minimal outline map of a
+defined area of land such as a continent, with optional labels at
+specified locations within the region, in the form of a recreation of the
+Monorail Map from The Simpsons with humorously oversized labelling (to
+imitate handwriting/scribbles) and sparsity of marked locations
+(which are all fictional).
+
+Specifically, it aims to recreate to best likeness using Cartopy
+the map of pre-Springfield Lyle Lanley Monorail locations from the
+iconic episode 'Marge vs. the Monorail' (1993) of the TV Series, as
+taken in likeness from the screen grab available at:
+https://simpsons.fandom.com/wiki/Brockway.
+
+"""
+
+import matplotlib.pyplot as plt
+from matplotlib.transforms import offset_copy
+
+import cartopy.crs as ccrs
+import cartopy.io.shapereader as shpreader
+from matplotlib.patches import Rectangle
+
+
+# Define choices for projection and locations to plot
+GEOM_PROJ = ccrs.PlateCarree()
+# Not real places, so locations pulled from location on map in still image
+# First value 2-tuple is the dot placemap location, second is where to write
+# the text label relative to that dot, to best match the map from the show.
+LOCATIONS_TO_PLOT = {
+    "Ogdenville": [(-111.8, 35.5), (1.5, -2.2), -6],
+    "North\nHaverbrook": [(-99.0, 43.5), (2.8, -0.5), -1],
+    "Brockway": [(-80.4, 33.6), (-3.4, -1.5), 3],
+}
+
+
+def main():
+    # Set up a plot with a light blue background and a white overall border.
+    # For proportions of land mass, font size and border to be as
+    # intended, need to keep 'figsize' and 'dpi' (4:3 ratio) as below.
+    fig = plt.figure(
+        figsize=(9, 7.5),
+        dpi=125,
+        facecolor="#AFCBBD",
+        edgecolor="white",  # sets up white border without need for another axes
+        linewidth=30,  # makes the border thicker as per original map
+    )
+    map_ax = fig.add_axes(
+        [0.035, 0.035, 0.93, 0.93], projection=ccrs.LambertConformal(), frameon=False
+    )
+
+    # Remove all axes ticks and labelling and center on location of USA
+    map_ax.set(xticks=[], yticks=[])
+    map_ax.set_extent([-120, -72.5, 20, 50], crs=ccrs.Geodetic())
+
+    # Plot only the USA landmass, in a fawn colour with a thin black border
+    shpfilename = shpreader.natural_earth(
+        resolution="110m", category="cultural", name="admin_0_countries"
+    )
+    countries = shpreader.Reader(shpfilename).records()
+    usa_border = [
+        country.geometry
+        for country in countries
+        if (country.attributes["NAME"] == "United States of America")
+    ]
+    map_ax.add_geometries(
+        usa_border,
+        GEOM_PROJ,
+        facecolor="#C39B6A",
+        edgecolor="black",
+    )
+
+    # Now add the location labels. The general approach is that covered in
+    # the 'Map tile acquisition' Catopy example, including the comment below.
```

Comment:
I think [annotate](https://matplotlib.org/stable/api/_as_gen/matplotlib.axes.Axes.annotate.html#matplotlib.axes.Axes.annotate) is the right tool for this job these days, so I have opened #2596 to update the existing example.

Note, you could use 'offset points' or 'offset fontsize' if you prefer one of those to 'offset pixels'.

Reply:
OK, so what should I do for my PR code, if anything?

Reply:
I recommend using `annotate`, as it should simplify the code (and not make use of the private `_as_mpl_transform` method).